tests/python_package_test/test_basic.py

```diff
@@ -515,3 +515,19 @@ def test_list_to_1d_numpy(y, dtype):
     result = lgb.basic.list_to_1d_numpy(y, dtype=dtype)
     assert result.size == 10
     assert result.dtype == dtype
+
+
+@pytest.mark.parametrize('init_score_type', ['array', 'dataframe', 'list'])
+def test_init_score_for_multiclass_classification(init_score_type):
+    init_score = [[0, 1, 2] for _ in range(10)]
```

Comment:
I'd prefer to have all unique values in the array to make it totally clear that order argument for `ravel()` is chosen correctly in the source code. I mean, when all values are unique, it's more intuitive that different orders will produce different results.
```suggestion
    init_score = [[i * 10 + j for j in range(3)] for i in range(10)]
```